codefresh.yml

```diff
@@ -0,0 +1,46 @@
+version: 1.0
+
+hooks:
+  on_elected:
+    exec:
+      image: quay.io/official-images/debian:buster
+      commands:
+        - '[ -n "$PUBLISH_BRANCH" ] || { echo "PUBLISH_BRANCH is empty" ; exit 1; }'
+        - '[ -n "$CF_BRANCH" ] || { echo "CF_BRANCH is empty" ; exit 1; }'
+        - '[ -n "$CF_REPO_NAME" ] || { echo "CF_REPO_NAME is empty" ; exit 1; }'
+        - '[ -n "$CF_REPO_OWNER" ] || { echo "CF_REPO_OWNER is empty" ; exit 1; }'
+
+stages:
+  - prep
+  - build
+  - publish
+
+steps:
+  clone:
+    stage: prep
+    type: git-clone
+    arguments:
+      repo: VIOOH/${{CF_REPO_NAME}}
+      revision: ${{CF_BRANCH}}
+      git: github
+
+  docker_build:
+    stage: build
+    type: build
+    arguments:
+      disable_push: true
+      image_name: apps/mwaa-local
```

Comment:
@praveengnsa it's a good question, I used `apps` because I knew that one would work, although tbh airflow is itself an "app" is it not? What is the intention of the "apps" namespace, is it meant for any apps, or only for "viooh" apps? Is this distinction of namespace even important in the context of ECR? I don't know...